src/com/dabomstew/pkrandom/newgui/NewRandomizerGUI.java

```diff
@@ -2555,6 +2562,7 @@ private void romLoaded() {
             ptRandomFollowEvolutionsRadioButton.setEnabled(true);
             ptRandomCompletelyRadioButton.setEnabled(true);
             ptFollowMegaEvosCheckBox.setVisible(romHandler.hasMegaEvolutions());
+            ptIsDualTypeCheckBox.setEnabled(true);
```

Comment:
This should probably be:
```suggestion
            ptIsDualTypeCheckBox.setEnabled(false);
```
With the current code, what happens is that when you initially load a ROM, the "Force Dual Types" checkbox is enabled, even though the Types radio button is "Unchanged"
![image](https://user-images.githubusercontent.com/12245827/154342325-a37756cd-2ba9-47b6-9f67-738445e3af16.png)